content/applications/sales/point_of_sale/overview/epos_ssc.rst

```diff
@@ -31,56 +29,182 @@ certificate is necessary to use your printer.
    - TM-P60II (Peeler: Wi-Fi® model)
    - TM-P80 (Wi-Fi® model)
 
-Generate a Self-signed certificate
-==================================
+To use the printer in Odoo, models that do not require an :doc:`IoT box
+<../../../productivity/iot/config/connect>` may need to establish a secure connection between their
+browser and the printer with :doc:`the HTTPS protocol <https>`. If so, trying to reach the printer
+IP address in HTTPS induces a warning page.
```

Comment:
```suggestion
IP address in HTTPS leads to a warning page.
```
Although correct, "induces" sounds really weird and out-of-place in this context